Source/WebKit/UIProcess/playstation/PageClientImpl.cpp

```diff
@@ -191,6 +201,13 @@ RefPtr<WebPopupMenuProxy> PageClientImpl::createPopupMenuProxy(WebPageProxy&  pa
     return { };
 }
 
+#if USE(GRAPHICS_LAYER_WC)
+bool PageClientImpl::usesOffscreenRendering() const
+{
+    return false;
```

Comment:
This relevant for PlayStation?

Reply:
It should be same with WinCairo. Or, should share the implementation.
usesOffscreenRendering is used for pixel dump of AC mode for WebKitTestRunner.

Reply:
WebKitTestRunner isn't enabled for PS in main so I'm going to defer an implementation at this point